src/screens/ShowDetailsScreen.tsx

```diff
@@ -1,13 +1,66 @@
+import { useEffect, useState } from 'react';
+import { useLocation } from 'react-router-dom';
+import { getMovieProviders } from '../helpers/getMovieUtils';
+import { MovieProviders, MovieDetailsData } from '../types/tmdb';
+
 /**
  * Screen to show more details of a specific show
  * Rendered after user clicks on show card
  * 
  * @returns {JSX.Element}
  */
 export default function ShowDetailsScreen(): JSX.Element {
+	const [providers, setProviders] = useState<MovieProviders>();
+	const location = useLocation();
+	console.log(location);
+
+	useEffect(() => {
+		const handler = async () => {
+			const data = await getMovieProviders(location.state.details.id);
+			setProviders(data) as unknown as MovieProviders;
+		};
+		handler();
+	}, []);
+
+	// TODO: Possibly create utility function 
+	const ratingHandler = (arr: MovieDetailsData): JSX.Element | null => {
+		for (let i = 0; i < arr.release_dates.results.length; i++) {
+			if (arr.release_dates.results[i].iso_3166_1 === 'US') {
+				return <p>{arr.release_dates.results[i].release_dates[0].certification}</p>;
+			}
+		}
+		return null;
+	};
+
 	return (
-		<>
-			<h1>Show Details Page</h1>
-		</>
+		<section>
+			{providers && (
+				<div className='flex'>
```

Comment:
Can we just flex the section instead of adding another div?

Reply:
Thought process was because this is a screen, and react fragments can't be styled - <section> would represent page, <div> would represent the details 'card'.

Reply:
I think this makes sense assuming we add a carousel or something below the details. As I was saying on Discord I would prefer to not add containers until they are needed, but your reasoning totally makes sense. And I think the plan is to add a carousel or other stuff below. 

As I am writing this, I also wonder if we would prefer to not render ONLY the providers section if no providers, rather than the entire details section being empty?